The Dropper Community Edition/map.xml

```diff
@@ -55,6 +56,7 @@
         <health>4</health>
         <effect duration="1s">regeneration</effect>
         <effect duration="oo">invisibility</effect>
+		<gamemode>adventure</gamemode>

```

Comment:
all of these `<gamemode>adventure</gamemode>` elements have incorrect spacing, it's like a tab inside